common-libs/common/src/services/RegistryResolveDidService.ts

```diff
@@ -0,0 +1,33 @@
+const RESOLVE_DID_PATH = '/api/v1/did/resolve-did'
+
+let fetch: any
+
+if (!fetch) {
+  fetch = require('node-fetch')
+}
+
+export class RegistryResolveDidService {
+  constructor(private readonly _registryUrl: string, private readonly _apiKey: string) {}
+
+  async resolveDid(did: string): Promise<any> {
+    const url = `${this._registryUrl}${RESOLVE_DID_PATH}`
```

Comment:
Please use `RegistryApiService` instead

Reply:
or higher-level `DidResolver`, which seems to be more appropriate here

Reply:
should be removed